### ISSUE TITLE
elevator not working properly disappearing floor and cloning floor

### ISSUE THREAD

**Issue by officerwolf** [McJty/RFTools#1702]
Issue description: elevator not working properly

Steps to reproduce: watch video for bug

Versions:
* Minecraft: 1.12.2
* Forge: 1.6.0
* CompatLayer (only if on Minecraft 1.10 or 1.11):
* McJtyLib:1.12-2.6.7
* RFTools:1.12-7.32

Possibly incompatible mods and versions: optifine 1.12.2 hd u d1
https://youtu.be/KOsU94Q9X2s for video

**Comment by josephcsible** [comment]
> Forge: 1.6.0

???

Also, anything in the logs?

**Comment by officerwolf** [comment]
let me check

**Comment by officerwolf** [comment]
Launcher
2.0.1003 
Wed, 25 Oct 2017 14:55:40 GMT  this what they are looking for?

**Comment by officerwolf** [comment]
found it forge-14.23.2.2637

**Comment by officerwolf** [comment]
nothing in logs besides this that i could see that would effect the elevator


23:34:37 | journeymap | JM-task-4 | warn | Unable to get any texture to use as color for rftools:screen[facing=west,horizontal_facing=west]
-- | -- | -- | -- | --

**Comment by josephcsible** [comment]
Can you see if you can reproduce this problem with just RFTools and its dependencies? If not, can you figure out what other mod is triggering it?

**Comment by officerwolf** [comment]
possibly ill try and replicate on a single player world

**Comment by officerwolf** [comment]
yes working with just rftools only now to figure out what mod is conflicting

**Comment by officerwolf** [comment]
figured out the problem for our server at least on a older version of rftools
if you have a chunk claimed for factions and with anti greiving it wouldnt allow blocks to move when we unclaimed a chuck/chunks it allowed blocks to be changed and now works . the thing is now to see if i can get rftools to work on the current version

**Comment by josephcsible** [comment]
What mod is doing the chunk claiming/anti-griefing?

**Comment by officerwolf** [comment]
Eagle factions

**Comment by josephcsible** [comment]
Ah, you're using Sponge. It's known to cause tons of bugs with all sorts of mods; is FTB Utilities a viable alternative for you?

Also, were you able to reproduce this with the latest version of RFTools?

**Comment by officerwolf** [comment]
As far as using a different mod for such case i dont know but i did try it with every mod on a single player world with nothing claimed which was the video i posted so yeah dont really know whats causing the issue between it

**Comment by josephcsible** [comment]
If you have SpongeForge but not EagleFactions, does it work? If so, can you report this to [EagleFactions' bug tracker](https://github.com/Aquerr/EagleFactions/issues)? If not, can you report this to [SpongeForge's bug tracker](https://github.com/SpongePowered/SpongeForge/issues)?
